### PR TITLE
Remove content from QPSES (PB) and (CS) schemas.

### PR DESCRIPTION
**Pull request by heofs**
### What is the context of this PR?
Remove text from schemas for QPSES(PB) and QPSES(CS).

### How to review 
See that the text "All functions of your organisation, for example, education, social services, and public administration" is gone.

### CODE REVIEW

**Review by rmccar — APPROVED** [review]
LGTM

File: data/en/qpses160_0002.json [hunk 1]
```diff
@@ -291,7 +291,7 @@
                                         "list": [
                                             "Agency workers paid directly from the agency payroll",
                                             "Self-employed workers",
-                                            "Voluntary Workers",
+                                            "Voluntary workers",
```
Comment (samiwel) [inline data/en/qpses160_0002.json]: This change is being made on a separate PR so can probably be removed from this branch.

**Review by samiwel — APPROVED** [review]
This branch will need rebasing after #2003 goes in and commits should be squashed, but otherwise changes look fine.

**Comment by SamGodwin2** [comment]
[Content to be removed from QPSES(PB) and QPSES(CS) surveys](https://trello.com/c/k8t2rqd8/952-content-to-be-removed-from-qpsespb-and-qpsescs-surveys)

**Pull request merged**